src/Main/Header.css

```diff
@@ -64,3 +64,84 @@
     padding-top: 40px;
   }
 }
+.character-selector {
+  position: relative;
+  height: 42px;
+  margin-bottom: 10px;
+}
+.character-selector>input, .character-selector>select, .character-selector>button, .character-selector>.realm-search {
+  display: inline-block;
+  position: absolute;
+}
+.character-selector .realm-search {
+  left: 102px;
```

Comment:
Because the dropdown doesn't like the flex-box layout, opening the dropdown then resulted in the whole header growing to the dropdowns size.
I'll try another approach, but can't promise that I'll find a cleaner solution for it

Reply:
The absolute positioning wasn't necessary with this library, seems like I left those from `react-select` and didn't bother to change it back to flexbox.
Is now flexbox-based again.